src/Testing/CodeHacking/CodeHacker.php

```diff
@@ -0,0 +1,320 @@
+<?php
+/**
+ * CodeHacker class file.
+ *
+ * @package WooCommerce/Testing
+ */
+
+//phpcs:disable Squiz.Commenting.FunctionComment.Missing, Squiz.Commenting.VariableComment.Missing
```

Comment:
This one helps with documentation, there's no reason to avoid including docblocks in functions, no code in WooCommerce is acceptable without proper inline docs.

Reply:
I hadn't included comments for this class because it's a stream wrapper class where most of the methods are irrelevant for the code hacking and are just transparent wrappers for the equivalent PHP filesystem functions. But if everything must have docs, then sure, I'll add them.

Reply:
...aaand done!